tests/test_git_utils.py

```diff
@@ -51,7 +51,7 @@ def test_git_cli_run_git_command_no_error(
 ):
     cli = GitCli()
 
-    working_directory = Path("TEST_DIR")
+    working_directory = Path(".")
```

Comment:
```suggestion
    working_directory = Path("TEST_DIR")
```

Again do not mix in changes like this.

Reply:
This change in tests is required because the relevant code now raises `FileNotFoundError` explicitly, meaning it fails if the directory does not exist. The change in the relevant code was necessary because `DOC501` and `DOC502` enforce explicit raising and documentation of exceptions.